src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -40,13 +40,15 @@ public void tearDown() {
   public void testSaveAndLoadUsers() throws PersistentDataStoreException {
     UUID idOne = UUID.randomUUID();
     String nameOne = "test_username_one";
+    String passwordOne = "password_one";
     Instant creationOne = Instant.ofEpochMilli(1000);
-    User inputUserOne = new User(idOne, nameOne, creationOne);
+    User inputUserOne = new User(idOne, nameOne, passwordOne, creationOne);
 
     UUID idTwo = UUID.randomUUID();
     String nameTwo = "test_username_two";
+    String passwordTwo = "password_two";
     Instant creationTwo = Instant.ofEpochMilli(2000);
-    User inputUserTwo = new User(idTwo, nameTwo, creationTwo);
+    User inputUserTwo = new User(idTwo, nameTwo, passwordTwo, creationTwo);
 
     // save
     persistentDataStore.writeThrough(inputUserOne);
```

Comment:
You can still add a test for the password without the storage layer.  Verify that the password of the loaded users is null.

This validates the intent of the current state of your code.  It also provides a nice place holder for actual loading password verification for the next step.